crates/router/src/types/domain/user.rs

```diff
@@ -1104,6 +1106,8 @@ pub struct NewUserRole<E: Clone> {
     pub created_at: PrimitiveDateTime,
     pub last_modified: PrimitiveDateTime,
     pub entity: E,
+    //TODO: This to be removed and a new type TenantLevel to be created
+    pub tenant_id: String,
```

Comment:
Add `tenant_id` in the level structs. This should not be added here.

Reply:
There is no need for tenant levels as of now. 

Unless we are not adding tenant level user roles.

Reply:
I thought of covering in the with the next PR, where we will give support for tenant levels user roles. We then should add tenant_id in all the level structs.